examples/quickstart/tutorial/hadoop/docker/Dockerfile

```diff
@@ -19,6 +19,11 @@ FROM centos:7
 
 USER root
 
+# CentOS is EOL, have to use vault
+RUN sed -i s/mirror.centos.org/vault.centos.org/g /etc/yum.repos.d/*.repo && \
+    sed -i s/^#.*baseurl=http/baseurl=http/g /etc/yum.repos.d/*.repo && \
+    sed -i s/^mirrorlist=http/#mirrorlist=http/g /etc/yum.repos.d/*.repo
+
```

Comment:
I think this warrants a separate PR with a clear explanation of what is being done here.
The rest of the changes look good.

Reply:
Remove the changes in this file.

Reply:
Raised separate here https://github.com/apache/druid/pull/16999.